packages/arb-token-bridge-ui/src/components/common/NetworkImage.tsx

```diff
@@ -0,0 +1,23 @@
+import Image from 'next/image'
+import { getNetworkLogo, getNetworkName, isNetwork } from '../../util/networks'
+
+export const NetworkImage = ({
+  chainId,
+  theme = 'light'
+}: {
+  chainId: number
+  theme?: 'light' | 'dark'
+}) => {
+  const size = isNetwork(chainId).isEthereum ? 12 : 16
+
+  return (
+    <div className="flex w-4 justify-center">
+      <Image
+        src={getNetworkLogo(chainId, theme)}
```

Comment:
seems like it's not properly cached
when user scrolls down to show more rows, the network images are being fetched and served on every new row

Reply:
removed network logo from the tx history - it's only a requirement for the reskin and will fix it there